libdokan/folderlist.go

```diff
@@ -86,7 +93,15 @@ func (fl *FolderList) open(ctx context.Context, oc *openContext, path []string)
 		}
 		fl.mu.Unlock()
 
+		h, er := libkbfs.ParseTlfHandle(
```

Comment:
Thanks @strib. I ended up changing `er` to more descriptive name, since `AddFavorite` takes `libkbfs.Favorite` which is derived from `h`.